documentation/notifications.md

```diff
@@ -18,13 +18,53 @@ Notifications are set up in your credentials JSON file. They will use the `notif
         ...
   } ,
   "notifications": {
-      "slack_url": "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX",
+      "slack_url": "https://api.slack.com/apps/2RFP2A7EN1/incoming-webhooks",
```

Comment:
Is `2RFP2A7EN1` something we should redact? If not, could we use a string that is more obviously fake?

Reply:
The Slack App ID `2RFP2A7EN1` is randomly generated based on their App ID syntax (10 characters, letters and numbers). If you think a more obviously fake Slack App ID makes sense, I'd be happy to adjust it.